docroot/sites/all/modules/features/registration_pages/registration_pages.features.conditional_fields.inc

```diff
@@ -150,10 +150,10 @@ function registration_pages_conditional_fields_default_fields() {
       ),
       'selector' => '',
       'values_set' => 1,
-      'value_form' => _registration_pages_get_entityform_id_from_type('ilr_alumni_association_event_reg'),
```

Comment:
Careful. Removing this function breaks the functionality. Because entityform ids are dynamic per environment, we need the function to get the value. You're commit here sets the value based on your local dev environment.